library/ssl_tls.c

```diff
@@ -5252,7 +5259,8 @@ const mbedtls_ssl_session *mbedtls_ssl_get_session_pointer(const mbedtls_ssl_con
          (SSL_SERIALIZED_SESSION_CONFIG_TRUNC_HMAC << \
              SSL_SERIALIZED_SESSION_CONFIG_TRUNC_HMAC_BIT) | \
          (SSL_SERIALIZED_SESSION_CONFIG_ETM << SSL_SERIALIZED_SESSION_CONFIG_ETM_BIT) | \
-         (SSL_SERIALIZED_SESSION_CONFIG_TICKET << SSL_SERIALIZED_SESSION_CONFIG_TICKET_BIT)))
+         (SSL_SERIALIZED_SESSION_CONFIG_TICKET << SSL_SERIALIZED_SESSION_CONFIG_TICKET_BIT) | \
+         (SSL_SERIALIZED_SESSION_KEEP_PEER_CRT << SSL_SERIALIZED_SESSION_CONFIG_KEEP_CRT_BIT)))
```

Comment:
Same naming nit as in the development PR: all the others have `CONFIG` in the name, and just add `_BIT` to specify the bit. This one doesn't have `CONFIG` in the name (but does in the `_BIT` value, but the `_BIT` value removes `_PEER`.

While this was probably done for length reasons, there is `SSL_SERIALIZED_SESSION_CONFIG_CLIENT_TICKET_BIT` which is as long as `SSL_SERIALIZED_SESSION_CONFIG_KEEP_PEER_CRT_BIT` would be